src/main/java/cz/codecamp/logger/loggers/MultiLogger.java

```diff
@@ -0,0 +1,23 @@
+package cz.codecamp.logger.loggers;
+
+import cz.codecamp.logger.LogLevelEnum;
+import cz.codecamp.logger.LoggerInterface;
+import cz.codecamp.logger.Message;
+import cz.codecamp.logger.PragmaticLoggerInterface;
+
+public class MultiLogger implements LoggerInterface, PragmaticLoggerInterface {
```

Comment:
`MultiLogger` nedědí od `LoggerAbstract`... když budu chtít odfiltrovat určitý druh zpráv z více loggerů naráz, budu muset nastavovat každému konkrétnímu loggeru zvlášť.